ledger-core/v2/insolar/dynamicrole_string.go

```diff
@@ -1,3 +1,8 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
 // Code generated by "stringer -type=DynamicRole"; DO NOT EDIT.
```

Comment:
this file should be deleted as there is no source for it. As far as I know we don't add copyright notice in generated code.

Reply:
Please, commit changes after we've fixed your IDE settings (probably use Shift+Shift -> Update Copyright...)

Reply:
see insolar/role.go